restful-tango/tangoREST.py

```diff
@@ -130,15 +139,14 @@ def createTangoMachine(self, image, vmms=Config.VMMS_NAME,
             disk=None,
             network=None)
 
-    def convertJobObj(self, dirName, jobObj):
+    def convertJobObj(self, key, courselab, jobObj):
```

Comment:
What's the reason for this change? It seems unrelated to job canceling.

Reply:
I got annoyed by the 1000 different ways of constructing the path so I refactored the code in this file a bit. I think I'll roll it back just so we're more assured that it works, haha.